diff --git a/data/org.gnome.Robots.appdata.xml.in b/data/org.gnome.Robots.appdata.xml.in
index 49052f1..e80e2f4 100644
--- a/data/org.gnome.Robots.appdata.xml.in
+++ b/data/org.gnome.Robots.appdata.xml.in
@@ -27,7 +27,7 @@
   </description>
   <screenshots>
     <screenshot type="default" width="1148" height="623">
-      <image>https://people.gnome.org/~mcatanzaro/gnome-robots.png</image>
+      <image>https://gitlab.gnome.org/GNOME/gnome-robots/-/raw/master/data/screenshot.png</image>
     </screenshot>
   </screenshots>
   <provides>
@@ -47,9 +47,10 @@
   <launchable type="desktop-id">org.gnome.Robots.desktop</launchable>
   <url type="homepage">https://wiki.gnome.org/Apps/Robots</url>
   <url type="bugtracker">https://gitlab.gnome.org/GNOME/gnome-robots/issues</url>
-  <url type="donation">http://www.gnome.org/friends/</url>
+  <url type="donation">https://www.gnome.org/donate/</url>
   <url type="help">https://help.gnome.org/users/gnome-robots/stable/</url>
-  <url type="translate">https://wiki.gnome.org/TranslationProject</url>
+  <url type="translate">https://l10n.gnome.org/module/gnome-robots/</url>
+  <url type="vcs-browser">https://gitlab.gnome.org/GNOME/gnome-robots</url>
   <update_contact>mcatanzaro_at_gnome.org</update_contact>
   <project_group>GNOME</project_group>
   <kudos>
@@ -60,26 +61,10 @@
   </kudos>
   <translation type="gettext">gnome-robots</translation>
   <developer_name>The GNOME Project</developer_name>
-  <content_rating type="oars-1.0">
+  <developer id="org.gnome">
+    <name>The GNOME Project</name>
+  </developer>
+  <content_rating type="oars-1.1">
     <content_attribute id="violence-cartoon">mild</content_attribute>
-    <content_attribute id="violence-fantasy">none</content_attribute>
-    <content_attribute id="violence-realistic">none</content_attribute>
-    <content_attribute id="violence-bloodshed">none</content_attribute>
-    <content_attribute id="violence-sexual">none</content_attribute>
-    <content_attribute id="drugs-alcohol">none</content_attribute>
-    <content_attribute id="drugs-narcotics">none</content_attribute>
-    <content_attribute id="drugs-tobacco">none</content_attribute>
-    <content_attribute id="sex-nudity">none</content_attribute>
-    <content_attribute id="sex-themes">none</content_attribute>
-    <content_attribute id="language-profanity">none</content_attribute>
-    <content_attribute id="language-humor">none</content_attribute>
-    <content_attribute id="language-discrimination">none</content_attribute>
-    <content_attribute id="social-chat">none</content_attribute>
-    <content_attribute id="social-info">none</content_attribute>
-    <content_attribute id="social-audio">none</content_attribute>
-    <content_attribute id="social-location">none</content_attribute>
-    <content_attribute id="social-contacts">none</content_attribute>
-    <content_attribute id="money-purchasing">none</content_attribute>
-    <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
 </component>
